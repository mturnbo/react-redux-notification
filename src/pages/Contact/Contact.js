import React from 'react';

const Contact = () => {
  return (
    <div>
      <h2>Contact Us</h2>
      <p>Spare ribs short ribs ham sirloin. Sausage andouille filet mignon frankfurter spare ribs sirloin buffalo t-bone hamburger drumstick strip steak brisket turducken jowl leberkas. Kielbasa meatball bacon tongue pork loin. Ham tri-tip pancetta, fatback ball tip andouille jowl porchetta ground round. Prosciutto bacon ham hock bresaola meatball chicken sirloin pork tongue burgdoggen pork chop. Rump kielbasa jowl chicken, filet mignon fatback buffalo meatball spare ribs beef tri-tip.</p>
    </div>
  );
};

export default Contact;
