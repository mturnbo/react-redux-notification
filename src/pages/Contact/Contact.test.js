import React from 'react';
import { mount } from 'enzyme';
import Contact from './index';

const wrapper = mount(<Contact />);

test('Contact Page Renders', () => {
  expect(wrapper).toMatchSnapshot();
});
