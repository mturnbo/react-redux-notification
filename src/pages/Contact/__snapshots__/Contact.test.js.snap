// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact Container Renders 1`] = `
<Contact>
  <div>
    <h2>
      Contact Us
    </h2>
    <p>
      Spare ribs short ribs ham sirloin. Sausage andouille filet mignon frankfurter spare ribs sirloin buffalo t-bone hamburger drumstick strip steak brisket turducken jowl leberkas. Kielbasa meatball bacon tongue pork loin. Ham tri-tip pancetta, fatback ball tip andouille jowl porchetta ground round. Prosciutto bacon ham hock bresaola meatball chicken sirloin pork tongue burgdoggen pork chop. Rump kielbasa jowl chicken, filet mignon fatback buffalo meatball spare ribs beef tri-tip.
    </p>
  </div>
</Contact>
`;

exports[`Contact Page Renders 1`] = `
<Contact>
  <div>
    <h2>
      Contact Us
    </h2>
    <p>
      Spare ribs short ribs ham sirloin. Sausage andouille filet mignon frankfurter spare ribs sirloin buffalo t-bone hamburger drumstick strip steak brisket turducken jowl leberkas. Kielbasa meatball bacon tongue pork loin. Ham tri-tip pancetta, fatback ball tip andouille jowl porchetta ground round. Prosciutto bacon ham hock bresaola meatball chicken sirloin pork tongue burgdoggen pork chop. Rump kielbasa jowl chicken, filet mignon fatback buffalo meatball spare ribs beef tri-tip.
    </p>
  </div>
</Contact>
`;
